staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -2108,6 +2108,9 @@ type ContainerPort struct {
 	// If specified, this must be an IANA_SVC_NAME and unique within the pod. Each
 	// named port in a pod must have a unique name. Name for the port that can be
 	// referred to by services.
+	// If a pod contains ports with duplicate names,
+	// when using services to reference ports by port name,
+	// the first duplicate port name will be referenced.
```

Comment:
I think we have to modify the previous sentence, amending more text makes it more confusing , I was thinking something like 

```
	// If specified, this must be an IANA_SVC_NAME . Each named port in a pod
        // must have a unique name, otherwise, if there are duplicate names, only the
	// first port name referred to by services will be chosen.
```

but wait for @danwinship and @thockin  opinion too

Reply:
seems better